src/AccountServices.js

```diff
@@ -0,0 +1,147 @@
+var RemoteRoom = require("matrix-appservice-bridge").RemoteRoom;
+var RemoteUser = require("matrix-appservice-bridge").RemoteUser;
+var Twitter    = require('twitter');
+var log        = require('npmlog');
+var OAuth      = require('oauth');
+var TwitterHandler = require('./TwitterHandler.js').TwitterHandler;
```

Comment:
Style: Some people like doing what you've done here (multi-line alignments for prettiness), and boy is it pretty. Don't do it.

The problem with this is that it adds needless maintenance burden when adding/removing variables which are aligned like this (you didn't even bother indenting `:1 ~ :5` to align it with this!). What's even worse is if you do decide to religiously do this, you obfuscate the diff when you add a new `var` (all lines will be modified).